x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/EnterpriseSearch.java

```diff
@@ -7,8 +7,6 @@
 
 package org.elasticsearch.xpack.application;
 
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
```

Comment:
Using org.elasticsearch.logging.* instead so we have one less module to require (org.apache.logging.log4j)